docs/immutability.md

```diff
@@ -0,0 +1,36 @@
+# Immutability
+
+We also provided a `@immutable` annotation, who works pretty much a constant, ideal to use on values objects, not mutable.
+
+A property setted as immutable can receive a value only a single time. Your value never can be changed.
```

Comment:
s/setted/marked
s/Your value never can be changed/The value can never be overwritten